packages/turf-helpers/lib/geojson.ts

```diff
@@ -57,189 +8,3 @@ export type BBox = BBox2d | BBox3d;
  * the Feature object with the name "id", and the value of this member is either a JSON string or number.
  */
 export type Id = string | number;
-
-/**
- * Position
- *
- * https://tools.ietf.org/html/rfc7946#section-3.1.1
- * Array should contain between two and three elements.
- * The previous GeoJSON specification allowed more elements (e.g., which could be used to represent M values),
- * but the current specification only allows X, Y, and (optionally) Z to be defined.
- */
-export type Position = number[]; // [number, number] | [number, number, number];
-
-/**
- * Properties
- *
- * https://tools.ietf.org/html/rfc7946#section-3.2
- * A Feature object has a member with the name "properties".
- * The value of the properties member is an object (any JSON object or a JSON null value).
- */
-export type Properties = { [name: string]: any } | null;
-
-/**
- * Geometries
- */
-export type Geometries =
-  | Point
-  | LineString
-  | Polygon
-  | MultiPoint
-  | MultiLineString
-  | MultiPolygon;
-
-/**
- * GeoJSON Object
- *
- * https://tools.ietf.org/html/rfc7946#section-3
- * The GeoJSON specification also allows [foreign members](https://tools.ietf.org/html/rfc7946#section-6.1)
- * Developers should use "&" type in TypeScript or extend the interface to add these foreign members.
- */
-export interface GeoJSONObject {
-  // Don't include foreign members directly into this type def.
-  // in order to preserve type safety.
-  // [key: string]: any;
-  /**
-   * Specifies the type of GeoJSON object.
-   */
-  type: string;
-  /**
-   * Bounding box of the coordinate range of the object's Geometries, Features, or Feature Collections.
-   * https://tools.ietf.org/html/rfc7946#section-5
-   */
-  bbox?: BBox;
-}
-
-/**
- * Geometry Object
- *
- * https://tools.ietf.org/html/rfc7946#section-3
- */
-export interface GeometryObject extends GeoJSONObject {
-  type: GeometryTypes;
-}
-
-/**
- * Geometry
- *
- * https://tools.ietf.org/html/rfc7946#section-3
- */
-export interface Geometry extends GeoJSONObject {
-  coordinates: Position | Position[] | Position[][] | Position[][][];
-}
-
-/**
- * Point Geometry Object
- *
- * https://tools.ietf.org/html/rfc7946#section-3.1.2
- */
```

Comment:
This Geometry type as-is, by extending GeoJSONObject, was not ensuring type is one of the 7 valid type values (as the spec requires), just that it's a string.  Geometry type under the `geojson` package now encompasses all 7 valid types.  This was one source of incompatibility between the two.